docs/security.html

```diff
@@ -273,6 +275,24 @@
     sasl.kerberos.service.name="kafka"</pre>
             </li>
         </ol></li>
+
+    <li><h4><a id="security_rolling_upgrade" href="#security_rolling_upgrade">Incorporating Security Features in a Running Cluster</a></h4>
+        You can secure a running cluster via one or more of the supported protocols discussed previously. This is done in two phases:
+        <ul>
+            <li>Incrementally bounce the cluster nodes to open an additional, secured port.</li>
```

Comment:
This still talks about opening an additional, secured port. This may be interpreted as if we may need multiple bounces to enable multiple ports (eg SSL and SASL). Supporting both SSL and SASL is not uncommon (SSL for machines and SASL for users), so I'd prefer if we made it clear that we could open multiple secured ports in this phase.